certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Lead Loader",
+    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple API calls that can create or get the items needed to automate the Lead creation process.",
+    "contact": {
+      "name": "Andre Roberts",
+      "url": "https://phiquest.com/",
+      "email": "andre@phiquest.io"
+    },
+    "version": "1.0"
+  },
+  "host": "xe8g8pkprc.execute-api.us-east-2.amazonaws.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [],
+  "paths": {
+    "/contact": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+            },
+          "200": {
+            "description": "200 status",
```

Comment:
@andreroberts1119 please provide a more detailed description for 2xx codes. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
Updated swagger file.